test/Microsoft.AspNet.Authentication.Test/Facebook/FacebookMiddlewareTests.cs

```diff
@@ -125,6 +125,67 @@ public class FacebookMiddlewareTests
         }
 
         [Fact]
+        public async Task NestedMapWillNotAffectRedirect()
```

Comment:
Ah yeah probably because  this was based of the map fix branch and I rebased to dev so its confused